### PR TITLE
Jorn-9000

### PR DESCRIPTION
**Pull request by Jorndel1**

### CODE REVIEW

File: 1_space_odyssey/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   
 <script src="js/mannuel-9000/app.js"></script>
  <!-- include your javascript file after my file below, leave mine in place. -->
-<script src="js/ben-9000/app.js"></script>
+<script src="js/jorn-9000/app.js"></script>
```
Comment (mannuelf) [inline 1_space_odyssey/index.html]: please put back ben-9000 file... add your file after his.  I want to have all your files imported by the time we are done ;-)

File: 1_space_odyssey/js/jorn-9000/app.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+
+/* 
+ 2. BROKEN SATELLITE::::::
+    You need to send the droid out on a space walk to fix the satellite, with out it you can't communicate with earth.
+    a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid 
+       cannot go outside, console log the correct log based on that condition. 
+*/
+if(hasJetPack)
+{
```
Comment (mannuelf) [inline 1_space_odyssey/js/jorn-9000/app.js]: please format / indent code 

````JS
if (someThing) {
   console.log("hello");
} else {
    console.log("good bye");
}
````
as per: https://github.com/mannuelf/clean-code-javascript 
have a look how all the if blocks are formatted throughout that document as a guide.

**Review by Jorndel1 — COMMENTED** [review]
Update from feedback

**Comment by mannuelf** [comment]
thank @Jorndel1 merging!! :-)

**Pull request merged**